File: 2264-largest-3-same-digit-number-in-string/2264-largest-3-same-digit-number-in-string.py
class Solution:
    def largestGoodInteger(self, num: str) -> str:
        res = ""
        for i in range(len(num) - 2):
            temp = num[i:i+3]
            if temp[0] == temp[1] and temp[1] == temp[2]:
                if temp > res:
                    res = temp
        return res

File: 2264-largest-3-same-digit-number-in-string/2264-largest-3-same-digit-number-in-string.cpp
class Solution {
public:
    string largestGoodInteger(string num) {
        string str = "";
        for (int i = 0; i < num.length() - 2; ++i) {
            string temp = num.substr(i, 3);
            if (temp[0] == temp[1] && temp[1] == temp[2]) {
                if (temp > str) {
                    str = temp;
                }
            }
        }
        return str;
    }
};